### PR TITLE
TBWR handler

### PR DESCRIPTION
**Pull request by qrort**

### CODE REVIEW

File: internal/backup_operations/make_backup.go [hunk 1]
```diff
@@ -203,3 +203,131 @@ func MakeBackup(
 
 	return backup, op, nil
 }
+
+func RetryBackup(
```
Comment (qrort) [inline internal/backup_operations/make_backup.go:207]: I think I will try to merge `MakeBackup` and `RetryBackup` into one method. Now they are the same, except one accepts MakeBackupRequest, while other accepts TBWROperation.
I can essentially make a separate type `MakeBackupInternalRequest` just for the method, transfer data to it and make the call.

Then, it'll have an additional argument `reuseBackupId`, which will not spawn another Backup entity for retries of TBWR op.

File: internal/handlers/take_backup_retry.go [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+		{
+			do, err := HandleTbOps(tbwr.RetryConfig, tbOps)
+			if err != nil {
+				tbwr.State = types.OperationStateError
+				tbwr.Message = err.Error()
+				errup := db.ExecuteUpsert(ctx, queries.NewWriteTableQuery().WithUpdateOperation(tbwr))
+				if errup != nil {
+					return errup
+				}
+				//I am kinda lost here. When should we return err and when should we xlog.Error()?
```
Comment (qrort) [inline internal/handlers/take_backup_retry.go:152]: help pls

File: internal/backup_operations/make_backup.go (hunk 1 above)
Comment (qrort) [reply internal/backup_operations/make_backup.go]: done

File: internal/handlers/take_backup_retry.go (hunk 2 above)
Comment (ulya-sidorina) [reply internal/handlers/take_backup_retry.go:161]: you should return error here, it should be logged by processor

**Review by ulya-sidorina — APPROVED** [review]

**Pull request merged**